governance/polls/Ratification Poll - Dissolution of Content Production, MKT-001, Multi-Sig Request (MIP47c3-SP1) - April 11, 2022.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Ratification Poll for Dissolution of Content Production, MKT-001, Multi-Sig Request (MIP47c3-SP1) - April 11, 2022
+summary: Regarding the offboard of the Content Production CU, this poll is for removing their wallet from the recognized multi-sig list.
```

Comment:
I propose an alternative wording: "This subproposals removes Content Production's (MKT-001) multi-sig from the MIP47 list of recognized MakerDAO multi-sigs"